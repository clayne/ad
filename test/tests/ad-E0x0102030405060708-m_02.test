ad | -c always -E0x0102030405060708 -m | endian.bin | | 0
