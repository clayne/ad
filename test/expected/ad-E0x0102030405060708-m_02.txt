[32m[K00000000000000C0[m[K[36m[K:[m[K [41;1m[K0102[m[K [41;1m[K0304[m[K [41;1m[K0506[m[K [41;1m[K0708[m[K [41;1m[K[m[KFF[41;1m[K01[m[K [41;1m[K0203[m[K [41;1m[K0405[m[K [41;1m[K0607[m[K  [41;1m[K........[m[K.[41;1m[K.......[m[K
[32m[K00000000000000D0[m[K[36m[K:[m[K [41;1m[K08[m[KFF FFFF FFFF FFFF FFFF [41;1m[K0102[m[K [41;1m[K0304[m[K [41;1m[K0506[m[K  [41;1m[K.[m[K.........[41;1m[K......[m[K
[32m[K00000000000000E0[m[K[36m[K:[m[K [41;1m[K0708[m[K [41;1m[K[m[KFFFF FFFF FFFF FFFF FF[41;1m[K01[m[K [41;1m[K0203[m[K [41;1m[K0405[m[K  [41;1m[K..[m[K.........[41;1m[K.....[m[K
[32m[K00000000000000F0[m[K[36m[K:[m[K [41;1m[K0607[m[K [41;1m[K08[m[KFF FFFF FFFF FFFF FFFF [41;1m[K0102[m[K [41;1m[K0304[m[K  [41;1m[K...[m[K.........[41;1m[K....[m[K
[32m[K0000000000000100[m[K[36m[K:[m[K [41;1m[K0506[m[K [41;1m[K0708[m[K [41;1m[K[m[KFFFF FFFF FFFF FFFF FF[41;1m[K01[m[K [41;1m[K0203[m[K  [41;1m[K....[m[K.........[41;1m[K...[m[K
[32m[K0000000000000110[m[K[36m[K:[m[K [41;1m[K0405[m[K [41;1m[K0607[m[K [41;1m[K08[m[KFF FFFF FFFF FFFF FFFF [41;1m[K0102[m[K  [41;1m[K.....[m[K.........[41;1m[K..[m[K
[32m[K0000000000000120[m[K[36m[K:[m[K [41;1m[K0304[m[K [41;1m[K0506[m[K [41;1m[K0708[m[K [41;1m[K[m[KFFFF FFFF FFFF FFFF FF[41;1m[K01[m[K  [41;1m[K......[m[K.........[41;1m[K.[m[K
[32m[K0000000000000130[m[K[36m[K:[m[K [41;1m[K0203[m[K [41;1m[K0405[m[K [41;1m[K0607[m[K [41;1m[K08[m[KFF FFFF FFFF FFFF FFFF  [41;1m[K.......[m[K.........
